guides/main_concepts/search.md

```diff
@@ -1,21 +1,21 @@
 # Search
 
-### Finding documents
+## Finding documents
 
 When the query input is received, MeiliSearch is building a more complex query taking into account *typos*, n-grams, and *synonyms* if configured.
 
 - _Typos_ - For example, if the query string is `botman`, MeiliSearch will return documents containing `batman`. [Read more about the typo rules](/guides/advanced_guides/typotolerance.md).
-- _N-grams_ - MeiliSearch is set to merge multi-words query into a single word. Ex: Searching for `bat mobile` will returns documents containing `batmobile`. Each words of the query will also be split in many ways so MeiliSearch can returns documents containing `new york` when querying for `newyork`.
+- _N-grams_ - MeiliSearch is set to merge multi-words query into a single word. Ex: Searching for `bat mobile` will returns documents containing `batmobile`. Each word of the query will also be split in many ways so MeiliSearch can returns documents containing `new york` when querying for `newyork`. [Read more about Concatenate and Split Queries](/guides/advanced_guides/concat.md)
```

Comment:
```suggestion
- _N-grams_ - MeiliSearch is set to merge multi-words query into a single word query. Ex: Searching for `bat mobile` will return documents containing `batmobile`. Each word of the query will also be split in many ways so MeiliSearch can returns documents containing `new york` when querying for `newyork`. [Read more about Concatenate and Split Queries](/guides/advanced_guides/concat.md)
```

Reply:
/!\ even if you disagree, be careful, I fix a grammar mistake at the same time.

Reply:
Yes, I think saying "query" again is redundant. And, it could also be potentially false.
Meilisearch will merge a multi-word query into a single word, amongst many other manipulations on the query words. Maybe this is the part that needs some clarification.
```suggestion
- _N-grams_ - Amongst all the different manipulations, MeiliSearch is set to merge multi-words queries into a single word. Ex: Searching for `bat mobile` will return documents containing `batmobile`. Each word of the query will also be split in many ways so MeiliSearch can returns documents containing `new york` when querying for `newyork`. [Read more about Concatenate and Split Queries](/guides/advanced_guides/concat.md)
```

Reply:
Still confusing for me. I prefer without `Amongst all the different manipulations`. It does not bring any information.
You can say `a single word one` if you don't like repetition.
And I dont understand why it's false, if a multi-word query exists, why a single-word query does not?

Reply:
It is false because if a multi-query word is transformed into a single-word query, this means that the new query is a single-word one. But this is not the case, both possibilities are used by MeiliSearch. Well, more than both because there are other manipulations done. Maybe this than : 
```suggestion
- _N-grams_ - MeiliSearch is set to merge multi-words queries into a single word when searching for matching documents. Ex: Searching for `bat mobile` will return documents containing `batmobile`. Each word of the query will also be split in many ways so MeiliSearch can returns documents containing `new york` when querying for `newyork`. [Read more about Concatenate and Split Queries](/guides/advanced_guides/concat.md)
```